public/car/index.html

```diff
@@ -25,29 +25,14 @@
         ></button>
       </div>
 
+      <div class="banner"><spec>Use the car ID in the car query to get more details about your EV</spec></div>
```

Comment:
Not really clear for me :) Perhps we could change the structure of the sentence a bit: To get more details about the car, please use `car(id: $carId)` query.

Reply:
What about this? 
To get more details about about a car, please use the car query: car(id: $carId)

Reply:
Top! You can use `<pre>car(id: $carId)</pre>` to make a code  like text :)